### DISCUSSION TITLE
Preparing build machine fails

### DISCUSSION THREAD

**Discussion by emappstore777** [codemagic-ci-cd/codemagic-docs#1963] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

Build from different repositories

### Expected results

Hello! In codemagic, while build starts - it fails on Preparing build machine with message:
Issues such as these are automatically reported back to us and don't consume any of your build minutes. Meanwhile, here’s what you can try:

Check the [GitHub Discussions](https://github.com/orgs/codemagic-ci-cd/discussions) page for information about possible incidents.
If the last command before the failure interacted with a 3rd party service, check their status page.
This might be a temporary glitch in our system, try to run a new build later.

Is anyone else has same issue now?

### Actual results

Hello! In codemagic, while build starts - it fails on Preparing build machine with message:
Issues such as these are automatically reported back to us and don't consume any of your build minutes. Meanwhile, here’s what you can try:

Check the [GitHub Discussions](https://github.com/orgs/codemagic-ci-cd/discussions) page for information about possible incidents.
If the last command before the failure interacted with a 3rd party service, check their status page.
This might be a temporary glitch in our system, try to run a new build later.

Is anyone else has same issue now?

**Comment by vivek2910** [comment]
Yes facing the same issue

**Reply by bevanchristian** [reply]
yes facing the same issue

**Reply by wisnhusurya** [reply]
yes facing the same issue

**Reply by LouisRaine** [reply]
facing the same issue

**Comment by kulakarla** [comment]
©emappstore777 @vivek2910 @bevanchristian 

Hello, we are aware of the issue and are investigating it.

**Comment by helinanever** [comment, marked as answer]
There's an ongoing incident, please see the announcement: https://github.com/orgs/codemagic-ci-cd/discussions/1965
We will post an update as soon as there is more information.
